src/types/generated/types.ts

```diff
@@ -2461,6 +2455,8 @@ export type CurrentCollectionOwnershipV2ViewVarianceFields = {
 
 /** columns and relationships of "current_collections_v2" */
 export type CurrentCollectionsV2 = {
+  /** An object relationship */
+  cdn_asset_uris?: Maybe<NftMetadataCrawlerParsedAssetUris>;
```

Comment:
Did we remove these on purpose earlier?

Reply:
No clue, this is just generated via pnpm run indexer-codegen and then fmt